### PR TITLE
GET /images/json respond w/ Created property + tests

### PR DESCRIPTION
**Pull request by cflynn07**

### CODE REVIEW

File: lib/middleware.js [hunk 1]
```diff
@@ -101,7 +101,8 @@ var middlewares = module.exports = {
         }
         var id = utils.randomId();
         images[id] = {
-          'id': id
+          'id': id,
+          'Created': new Date().valueOf / 1000 | 0
```
Comment (bkendall) [inline lib/middleware.js]: I think you're missing the `()` for `valueOf`

File: test/docker.js [hunk 2]
```diff
@@ -433,6 +433,7 @@ describe('images', function () {
         expect(images).to.have.length(2);
         expect(images[0].RepoTags).to.have.length(1);
         expect(images[0].RepoTags[0]).to.equal('testImage:latest');
+        expect(images[0].Created).to.be.a.number();
```
Comment (bkendall) [inline test/docker.js]: you can use `.to.be.about(new Date().valueOf() / 1000 | 0, 10);` here (which, from my quick understanding, would be "within ten seconds")

https://github.com/hapijs/code#aboutvalue-delta

Comment (cflynn07) [inline test/docker.js]: added this check

**Comment by bkendall** [comment]
LGTM!

**Pull request merged**